cases/file_to_s3_reliability/terraform/main.tf

```diff
@@ -0,0 +1,70 @@
+provider "aws" {
+  region  = "us-east-1"
+  version = "~> 2.53"
+}
+
+terraform {
+  required_version = ">= 0.12"
+  backend "s3" {}
+}
+
+module "topology" {
+  source = "../../../terraform/aws_uni_topology"
+
+  providers = {
+    aws = aws
+  }
+
+  pub_key                 = var.pub_key
+  subject_instance_type   = var.subject_instance_type
+  test_configuration      = var.test_configuration
+  test_name               = var.test_name
+  user_id                 = var.user_id
+  results_s3_bucket_name  = var.results_s3_bucket_name
+}
+
+resource "aws_s3_bucket" "logs-bucket" {
+  # data is namespaced by host within the bucket
+  bucket = "file-to-s3-reliability-test-data"
```

Comment:
Again, should be templated.

Reply:
Yeah, I would template this so it's namespaced like our instance names. Ex:

```
vector-test-${var.user_id}-${var.test_name}-${var.test_configuration}-test-data
```